ansible/TestbedProcessing.py

```diff
@@ -377,45 +388,143 @@ def makeLab(data, devices, testbed, outfile):
+                            ansible_host = dev.get("ansible").get("ansible_host")
                             entry += "\tansible_host=" + ansible_host.split("/")[0]
                         except:
                             print("\t\t" + host + ": ansible_host not found")
 
                         if ansible_host:
                             try: # get ansible ssh username
-                                ansible_ssh_user = devices.get(host.lower()).get("ansible").get("ansible_ssh_user")
+                                ansible_ssh_user = dev.get("ansible").get("ansible_ssh_user")
                                 entry += "\tansible_ssh_user=" + ansible_ssh_user
                             except:
                                 print("\t\t" + host + ": ansible_ssh_user not found")
 
                             try: # get ansible ssh pass
-                                ansible_ssh_pass = devices.get(host.lower()).get("ansible").get("ansible_ssh_pass")
+                                ansible_ssh_pass = dev.get("ansible").get("ansible_ssh_pass")
                                 entry += "\tansible_ssh_pass=" + ansible_ssh_pass
                             except:
                                 print("\t\t" + host + ": ansible_ssh_pass not found")
+                        try: #get hwsku
+                            hwsku = dev.get("hwsku")
+                            if hwsku is not None:
+                               entry += "\thwsku=" + hwsku
+                        except AttributeError:
+                            print("\t\t" + host + ": hwsku not found")
+
+                        try: #get card_type
+                            card_type = dev.get("card_type")
+                            if card_type is not None:
+                               entry += "\tcard_type=" + card_type
+                        except AttributeError:
+                           print("\t\t" + host + ": card_type not found")
+
+                        try: #get num_fabric_asics
+                            num_fabric_asics = dev.get("num_fabric_asics")
+                            if num_fabric_asics is not None:
+                               entry += "\tnum_fabric_asics=" + str( num_fabric_asics )
+                        except AttributeError:
+                            print("\t\t" + host + " num_fabric_asics not found")
+
+                        try: #get num_asics
+                            num_asics = dev.get("num_asics")
+                            if num_asics is not None:
+                               entry += "\tnum_asics=" + str( num_asics )
+                        except:
+                            print("\t\t" + host + " num_asics not found")
+
+                        if card_type != 'supervisor':
+                           entry += "\tstart_switchid=" + str( start_switchid )
+                           if num_asic is not None:
```

Comment:
right now the inputs are
1. start_switchid
2. default assumption that num_cores is 1

Apart from these two inputs, num_asics can also be an input to TestbedProcessing so that, it can generate the 6,8,10 

So that we can make the math done by the TestbedProcessing instead of passing as inputing a list with manual calculations